### DISCUSSION TITLE
менеджер построек "ручных" производств

### DISCUSSION THREAD

**Discussion by Eirev** [fedorovvl/tso_client#21] [Ideas]
сразу уточняю, это не запрос и не пожелание. 
это идея которая может быть когда-нибудь пригодится.

в игре накопилось большое количество строений, производство в которых запускается в ручном режиме: 
ум, 
умр, 
маслобойня, 
маслозавод, 
шоколадная фабрика, 
статуя возлюбленных, 
прилавок с закусками, 
лаборатория, 
несколько видов деревьев (производство в каждом виде запускается отдельно), 
возможно что-то еще забыл, например переплетчик приключений и лазарет, но там все делается очень долго

было бы удобным получить возможность по нажатию одной кнопки видеть, не простаивает ли какое-либо из этих строений. 

может быть актуально, если выпадаешь из игры на какое-то время и зайдя, начинаешь наводить порядки и забываешь, что часть давно выработалась и ничего не производит.

опционально при нажатии неплохо бы видеть время загрузки в форме:
ум: пусто
умр: загрузка менее чем на 1 сутки
дерево (зеленое): загружено менее чем на 7 суток
дерево (синее): загружено более чем на 30 суток

**Comment by fedorovvl** [comment]
реализовано в 1.5.3
